runtime/include/chpl-external-array.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2004-2018 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ * 
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * 
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _chplexternarray_H_
+#define _chplexternarray_H_
```

Comment:
this guard does not actually match the filename

Reply:
Based on my experience with hyphens in C names, I got scared so I avoided using them...

Reply:
it's not the hyphens - I would have expected it to be _chpl_external_array_H_

There is 'external' vs 'extern'

Reply:
Oooooohhhhhhh